src/utils/oauth/index.ts

```diff
@@ -0,0 +1,18 @@
+//#region > Imports
+//> RSA 
+// Contains authentication service
+import RSA from "react-very-simple-oauth";
+//> GithubProvider
+// Contains the github authentication provider
+import GithubProvider from './GithubProvider';
+//#endregion
+
+//#region > Exports
+export default RSA;
+export { GithubProvider };
+//#endregion
+
+/**
+ * SPDX-License-Identifier: (EUPL-1.2)
+ * Copyright © Simon Prast
```

Comment:
This copyright tag is outdated.
Please change it to:  "Copyright © 2019-2020 Simon Prast"